torch/_tensor_str.py

```diff
@@ -28,8 +29,8 @@ def set_printoptions(
     Args:
         precision: Number of digits of precision for floating point output
             (default = 8).
-        threshold: Total number of array elements which trigger summarization
-            rather than full `repr` (default = 1000).
+        threshold: Upper bound of dim size for a full representation. Dims with
```

Comment:
Is this sensible?  This means if I have say a size [500, 500, 500] tensor it will print 500^3 elements?  Numpy prints something like this instantly, but I'm still waiting for torch to print it...

What is the motivation behind this part of the change?  Our old printing of large tensors seemed more or less fine (minus the things you improved in say, vector printing that could be applied generally).

Reply:
Hm, if it takes a long time to print large tensors then I think we should change it back to limit by elements

Reply:
That seems good; you can just minimize the number of changes I think.